core/src/com/fallenflame/game/LevelController.java

```diff
@@ -640,12 +640,13 @@ public void update(float dt) {
         // If victorious, mark victorious
         if (player.hasWon()) setLevelState(LevelState.WIN);
 
-        // If dying or dead, that's it. Don't update anything else.
+        // If player won or is dead, that's it. Don't update anything else.
         // (such as light, fog, enemies, etc)
-        if (player.isDead() || player.isDying() || player.hasWon()) return;
+        if (player.isDead() || player.hasWon()) return;
 
-        if (player.isWinning()) {
-            player.setLightRadius(player.getLightRadiusSprint()); //increase light radius to see fire buddy
+        //Update nothing except lights
+        if (player.isWinning() || player.isDying()) {
```

Comment:
If this happens when they're dying, you've now introduced a huge benefit to them, as they can see more as they die/get a lot more of their surrounding than intended